clients/client-java/src/test/java/com/datastrato/gravitino/client/TestGravitinoMetalake.java

```diff
@@ -126,7 +126,7 @@ public void testLoadCatalog() throws JsonProcessingException {
         new CatalogDTO.Builder()
             .withName("mock")
             .withComment("comment")
-            .withType(Catalog.Type.FILE)
+            .withType(Catalog.Type.STREAM)
```

Comment:
I suggest you add a new `CatalogDTO` instance instead to change it from `STREAM` to `FILE`.

Reply:
@yuqi1129 Does that mean we have to introduce a new `Catalog.Type` like `UNKNOWN`?

Reply:
I think it is not necessary to add a "UNKONW" type, the current change is OK for me. We can refactor the UT later on.

Reply:
OK